internal/sirius/api_client.go

```diff
@@ -103,9 +103,9 @@ func (c *ApiClient) newRequest(ctx Context, method, path string, body io.Reader)
 }
 
 func (c *ApiClient) logErrorRequest(req *http.Request, err error) {
-	c.logger.Print("method: " + req.Method + ", url: " + req.URL.Path)
+	c.logger.Info("method: " + req.Method + ", url: " + req.URL.Path)
 	if err != nil {
-		c.logger.Print(err)
+		c.logger.Error(err.Error())
```

Comment:
I'm not too familiar with this repo, but this change _might_ result in triggering alarms for 4xx errors. You could check locally if it's logging `{"level":"ERROR", ...}` with a 4xx-type message.

Reply:
Could be wrong but the only difference I could see between logger.Info() and logger.Error() was that it outputs "level":"INFO" / ,"level":"ERROR". And as the 'level' attribute is stripped out again by your slog.HandlerOptions, the output is actually the same for both. 

Begs the question, do we _want_ to strip out the logging level from the output?